pkg/issuer/const.go

```diff
@@ -11,14 +11,16 @@ const (
 	// Default to 30 days if Issuer.spec.renewBefore is not set
 	RenewCertificateBeforeDuration = time.Hour * 24 * 30
 	// Minimum duration before renewal.
-	MinimumCertificateValidityDuration = time.Minute * 5
+	MinimumCertificateRenewalWindowDuration = time.Minute * 5
```

Comment:
I'm confused about this parameter being renamed - previously, it made sense that it was the minimum period of time a cert can be valid for.

Now, I'm not so sure 🤔 there are two things that can be configured, thus two things that can have minimums:

* The certificate validity duration
* The amount of time before the expiry date before we begin attempting to renew a cert

I'd say a sensible minimum validity duration is 1h for now.
Let's say the minimum amount of time before cert expiry that we'll begin attempting renewal is 5 minutes.

We can review these minimums in future 😄 

We also need to set default values though, and those defaults should be fairly more liberal than those minimums. Happy to keep those as you've specified/similar to other issuers.